decoder_test.go

```diff
@@ -330,6 +370,56 @@ func TestDecodeColumn(t *testing.T) {
 	}
 }
 
+// It should only contain test cases which can't be tested in TestDecodeColumn
+func TestDecodeColumnGCV(t *testing.T) {
```

Comment:
I think we don't need to create a separate test function because `spanner.NewRow` accepts the `spanner.GenericColumnValue` as well.

See: https://github.com/googleapis/google-cloud-go/blob/633dc86f615ee32c1ad1ec704bf6bc858a4d535c/spanner/value.go#L4424-L4428

Reply:
I think that I don't want to test `spanner.NewRow` and `(*spanner.Row).Column`, so `TestDecodeColumnGCV` is more clean unit tests.
But I don't want to rewrite all existing test cases in `TestDecodeColumn` to equivalent hand written `spanner.GenericColumnValue`, so I agree to merge new test cases into `TestDecodeColumn`.